### PR TITLE
Allow variable substitution with ${workspaceFolder} in libPaths config.

### PR DESCRIPTION
**Pull request by drumlab**
This allows us to add workspace relative paths to libPaths.

### CODE REVIEW

**Comment by drumlab** [comment]
@navrocky Hello, is this something can be accepted? Thanks!

**Comment by ProfMoo** [comment]
@navrocky This would be very helpful for me. Would love if this could be reviewed and merged.

**Comment by BronzeDeer** [comment]
I'm not sure if hardcoding the single variable (especially exactly and not allowing for whitespace differences) etc is the best reason, wouldn't a better solution be to put the current directory always at workspace root instead of next to the file? Currently, static analysis seems to already  use that as current path and only fumbles when running a file for preview.
(Try it, put the correct relative libPath in, and you will only get runtime errors, but if you remove it you will get many "not found" warnings as well)

File: server/server.ts [hunk 1]
```diff
@@ -133,9 +136,14 @@ export const initializer = (
...
 }
 
+const expandPathVariables = (path: string): string => {
+  return path.replace(/\${workspaceFolder}/g, workspaceFolder);
+}
+
 export const configUpdateProvider = (
   change: server.DidChangeConfigurationParams,
 ): void => {
+  console.log(change);
```
Comment (navrocky) [inline server/server.ts]: You need to remove console.log

**Review by navrocky — CHANGES_REQUESTED** [review]

File: server/server.ts (hunk 1 above)
Comment (drumlab) [reply server/server.ts]: Done.

**Comment by drumlab** [comment]
I somehow missed the review earlier. Thanks!

**Comment by ProfMoo** [comment]
@navrocky Thanks for the merge! Can we get a new version tagged and released? I want to update my version used in VSCode

**Comment by navrocky** [comment]
> @navrocky Thanks for the merge! Can we get a new version tagged and released? I want to update my version used in VSCode

Just publish it as 1.2.1

**Pull request merged**